### DISCUSSION TITLE
openmptcprouter-vps-admin

### DISCUSSION THREAD

**Discussion by yannick2801** [Ysurac/openmptcprouter#2984] [General]
What is the purpose of OpenMPTCrrouter-VPS-admin? 
Does it allow you to manage the routers via an interface?
How to install it on the VPS?

[https://github.com/Ysurac/openmptcprouter-vps-admin](url)

**Comment by Ysurac** [comment]
OpenMPTCProuter-VPS-Admin is the API part of the VPS. It's installed via the VPS script part.
It's used to retrieve the configuration for the router and for the router to send changes to the VPS.
This doesn't allow to manage the router via an interface.

**Reply by yannick2801** [reply]
Thanks for your answer.

is there some example of API that can be used ? 

BR
Yannick

**Reply by Ysurac** [reply]
No documentation publicly available for now. But you can check the code: it's used by "openmptcprouter-vps" script to send/retrieve config and by the status page to display traffic usage and VPS status.

**Reply by yannick2801** [reply]
ok thanks.

BR
Yannick
